cram_knowrob/cram_semantic_map_utils/src/mesh-list.lisp

```diff
@@ -0,0 +1,130 @@
+(in-package :sem-map-utils)
```

Comment:
This package is not the right place to put these very robcog-specific things. It should be at least in your cram_kr_vr package or something.